docs/contributing/guidelines-for-writing-documentation.md

```diff
@@ -2,7 +2,25 @@
 
 * Documentation is distributed with the source code of Shopsys Framework inside [docs](https://github.com/shopsys/shopsys/tree/7.3/docs/) directory in the project root.
 * Documentation is written in [Markdown format](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+* Output HTML is rendered from the `*.md` files using [mkdocs](https://www.mkdocs.org/) and hosted on [readthedocs](https://readthedocs.org/).
+    * The docs live at [https://docs.shopsys.com](https://docs.shopsys.com).
+    * When using Docker, you can see the rendered docs locally on `http://127.0.0.1:1300`.
+        * See `mkdocs` container defined in `docker-compose.yml`.
+    * You can even access the rendered docs under `/documentation/` sub-folder for any branch that is built on our Jenkins CI server.
+    * The main [`mkdocs.yml`](https://github.com/shopsys/shopsys/blob/7.3/mkdocs.yml) configuration file is located in the monorepo root.
 * All documentation files should be named same as first heading (in lowercase, non-alphanumeric characters replaced by dash).
-* References to project files and classes have to be absolute links to the GitHub in proper version - eg. `[app/config/config.yml](https://github.com/shopsys/shopsys/blob/7.3/project-base/app/config/config.yml)`.
+* References to project files and classes have to be absolute links to the GitHub in proper version - e.g. `[app/config/config.yml](https://github.com/shopsys/shopsys/blob/7.3/project-base/app/config/config.yml)`.
```

Comment:
If we have to use proper version, is there any check/fix that will update version with new release?

Reply:
I agree there should be some check or fix, however, I am not sure about the proper solution - I have created an issue for that - https://github.com/shopsys/shopsys/issues/1442

Reply:
So you are suggesting that during release process someone have to go through all the docs and change the links to rpoper version - e.g. 
`[app/config/config.yml](https://github.com/shopsys/shopsys/blob/7.3/project-base/app/config/config.yml)`.
->
`[app/config/config.yml](https://github.com/shopsys/shopsys/blob/8.0/project-base/app/config/config.yml)`.

Reply:
yes, this is something you need to do when merging a branch to higher branch - but this not a new rule :wink:

Reply:
I have also created a PR that adds the instruction to release process - https://github.com/shopsys/shopsys/pull/1467